packages/block-editor/src/components/block-tools/index.js

```diff
@@ -152,6 +153,15 @@ export default function BlockTools( {
 				// block so that focus is directed back to the beginning of the selection.
 				// In effect, to the user this feels like deselecting the multi-selection.
 				selectBlock( clientIds[ 0 ] );
+			} else if ( clientIds.length === 1 ) {
+				event.preventDefault();
+				clearSelectedBlock();
+				// This is temporary code to see if this works with NVDA. I'm not sure the best way to find the region that contains the block editor.
+				document
+					.querySelector(
+						'[role="region"][aria-label="Editor content"]'
+					)
+					.focus();
```

Comment:
Hah, this does what I described here no? https://github.com/WordPress/gutenberg/pull/62290#discussion_r1630701078
Doesn't it fix that issue too?

Reply:
Maybe instead of focussing right here and having to query the DOM, you could listen for block selection clearing (an effect checking for no selection) in writing flow?

Reply:
Ooooh - that could work well! That would fix any instance where block selection is cleared. It's still an issue of _where do we put focus_? Maybe we can add an extra element with an aria label and `tabindex=-1` that wraps the block list?